### PR TITLE
Add several LS completion/rename support improvements to release note

### PR DESCRIPTION
**Pull request by IMS94**
## Purpose
$subject

### CODE REVIEW

File: release-notes/swan-lake-alpha3-release-note.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
 #### Developer Tools
 
 ##### Language Server
+###### Completions
+- Completion support for field accesses, varible declarations and assignment expressions are now context aware. i.e:
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha3-release-note.md:124]:
```suggestion
- Completion support for field access, variable declarations, and assignment expressions are now context aware. i.e: 
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 2]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
 #### Developer Tools
 
 ##### Language Server
+###### Completions
+- Completion support for field accesses, varible declarations and assignment expressions are now context aware. i.e: 
+  - When we declara a vairiable or perform an assignment, suggestions show the methods/variables which return/are values of same type as the variable.
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha3-release-note.md:125]:
```suggestion
  - When we declare a variable or perform an assignment, suggestions show the variables which are of same type as the variable and methods that return a value of the same type.
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 3]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
...
+  - Object and record field accesses now suggest object/record fields sorted at the top.
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha3-release-note.md:126]: How about
```suggestion
  - Object and record field access now suggests object and record fields first.
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 4]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
...
+###### Rename
+- Rename now validates the `newText` for a valid identifier before performing the rename
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha3-release-note.md:128]:
```suggestion
- Rename now validates if the new name is a valid identifier before performing the rename.
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 5]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
 #### Developer Tools
 
 ##### Language Server
+###### Completions
+- Completion support for field access, variable declarations, and assignment expressions are now context aware. i.e: 
+  - When we declare a variable or perform an assignment, suggestions show the variables which are of same type as the variable and methods that return a value of the same type.
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha3-release-note.md:125]:
```suggestion
  - When we declare a variable or perform an assignment, suggestions show the variables which are of same type as the variable and functions that return a value of the same type.
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 6]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
 #### Developer Tools
 
 ##### Language Server
+###### Completions
+- Completion support for field access, variable declarations, and assignment expressions are now context aware. i.e:
```
Comment (praneesha) [inline release-notes/swan-lake-alpha3-release-note.md:124]:
```suggestion
- Completion support for field access, variable declarations, and assignment expressions are now context-aware. That means the following.
```

File: release-notes/swan-lake-alpha3-release-note.md (hunk 5 above)
Comment (praneesha) [inline release-notes/swan-lake-alpha3-release-note.md:125]:
```suggestion
  - When we declare a variable or perform an assignment, suggestions show the variables, which are of the same type as the variable, and methods that return a value of the same type.
```

File: release-notes/swan-lake-alpha3-release-note.md [hunk 7]
```diff
@@ -120,6 +120,12 @@ io:println(nameText); // "DanBrown"
 #### Developer Tools
 
 ##### Language Server
+###### Completions
+- Completion support for field access, variable declarations, and assignment expressions are now context aware. i.e: 
+  - When we declare a variable or perform an assignment, suggestions show the variables which are of same type as the variable and methods that return a value of the same type.
+  - Object and record field access now suggests object and record fields first.
```
Comment (praneesha) [inline release-notes/swan-lake-alpha3-release-note.md:126]:
```suggestion
  - Object and record field access now suggests the object and record fields first.
```

**Comment by IMS94** [comment]
This PR is not required anymore as the list of bugs fixed are now included in the release note under each section. For language server the [list](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A"Ballerina+Swan+Lake+-+Alpha3"+label%3AType%2FBug+label%3ATeam%2FLanguageServer) is included which contains the issues related to this PR. Shall I close this? @praneesha @MaryamZi

**Pull request closed without merging**